src/RegionSelect.js

```diff
@@ -0,0 +1,22 @@
+import Select from 'react-select'
+import { customStyles } from './selectStyling';
+
+function RegionSelect({ data, regionSelect }) {
+
+    const regionData = data;
+    const regionNames = regionData.map((network, index) => ({ value: `${network.id}`, label: `${network.name}` }));
```

Comment:
i dont see index used in this map function, and i think index is optional - small thing but i think you can leave index out in this case

Reply:
Removed Index. Thanks!